Lombiq.Privacy/Manifest.cs

```diff
@@ -5,7 +5,7 @@
     Name = "Lombiq Privacy",
     Author = "Lombiq Technologies",
     Website = "https://lombiq.com",
-    Version = "3.0.1"
+    Version = "3.1.0-alpha"
```

Comment:
Next time don't skip a minor version when publishing an alpha (the current version at the time was 3.0.1 so this should have been 3.0.2-alpha): https://lombiq.atlassian.net/wiki/spaces/DEV/pages/786857987/Managing+releases+of+open-source+projects#:~:text=When%20publishing%20a%20pre%2Drelease%20version%20for%20something%20temporary%20you%20do%20under%20an%20issue%2C%20then%20always%20use%20the%20next%20patch%20version%20with%20a%20suffix.%20E.g.%20if%20the%20current%20latest%20released%20version%20if%201.2.3%20then%20publish%201.2.4%2Dalpha.0.